File: interface.py
from repository import codigos

while True:
    codigo_ddd = int(input("Digite o DDD:"))
    found = False
    for ddd in codigos:
        if ddd["codigo"] == codigo_ddd:
            print("A cidade cujo DDD é", codigo_ddd, "é", ddd["cidade"])
            found = True
            break

    if not found:
        print("DDD nao cadastrado")

File: DS2 em JS/interface.js
let loop = true;

while (loop) {
  const ddd = parseInt(prompt("Digite o DDD: "));

  if (codigos.has(ddd)) {
    alert("a cidade de DDD" + ddd + " é " + codigos.get(ddd));
  } else {
    alert("DDD não cadastrado");
  }

  loop = confirm("Quer digitar outro código?");
}